### DISCUSSION TITLE
All Data Type not shown in SQLite Studio 3.4.0

### DISCUSSION THREAD

**Discussion by jsdpr15** [pawelsalawa/sqlitestudio#4248] [Q&A]
### Details
Hey Sir,

I am Using Latest build created by u (SQLite Studio 3.4.0). this version doesn't contain all data types
see below image.
please check
![image](https://user-images.githubusercontent.com/63514848/138978850-40a39b11-8bdb-46bd-83a4-fa9c3ebd5c8b.png)

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
Windows 11/ 10 Pro 64-bit

### SQLiteStudio version
SQLite Studio 3.4.0 Latest Build
Windows 64-bit release build #12

**Comment by pawelsalawa** [comment]
It is intentional. See #3769

**Comment by jsdpr15** [comment]
So sir, how to fix this issue please help

**Comment by pawelsalawa** [comment]
It's not an issue. You still can type in any data type you want. It's just that it's not in the dropdown, but you can still enter your own type, whatever you like.

**Reply by edrobinson** [reply]
This is a PITA! Whose idea was this?

**Comment by crippled-truckie** [comment]
I see the data types are all in the tools/configuration but the ones I use most don't show up in the data types in the editor I am getting sick of having to type in the common data types that used to be in the older versions, can you please reverse this and I think it's not very nyuitive as the older version

**Comment by rec1968** [comment]
Those complaining about the _change_ might find it wise to study `https://www.sqlite.org/datatype3.html` which explains **Data Types in SQLite.** Since the underlying database engine is SQLite, an understanding of the way it handles datatypes might be useful and possibly save them from unexpected problems with their databases.
